README.rst

```diff
@@ -74,11 +68,140 @@ To install in a virtual environment in your current project:
 
 
 
-Usage Example
-=============
+Usage Examples
+==============
+
+See `examples/ble_file_transfer_simpletest.py` for a client example. A stub server implementation is in `examples/ble_file_transfer_simpletest.py`.
+
+Protocol
+=========
+
+The file transfer protocol is meant to be simple and easy to implement. It uses free space counts as a way to rate limit file content data transfer. All multi-byte numbers are encoded with the least significant byte first ("<" in CPython's struct module).
+
+GATT Service
+--------------
+
+The UUID of the service is `0xfebb`, Adafruit's 16-bit service UUID.
```

Comment:
The adafruit-ble-adafruit services uses a 128-bit service UUID. I know we have the one 16-bit one, but could you use a 128-bit one?

Reply:
We could use a 128-bit one but I think we'll use this a lot so the 16bit will be useful.